main.go

```diff
@@ -133,6 +137,7 @@ func printError(err error) {
 }
 
 func main() {
+	rand.Seed(randSeed)
```

Comment:
```go
rand.Seed(time.Now().UnixNano())
```

And then get rid of the constant.

Otherwise every run of the program will generate the very same numbers...